modules/fileout_netcdf/FONcArray.cc

```diff
@@ -731,6 +734,15 @@ void FONcArray::define(int ncid) {
                         FONcUtils::handle_error(stax, err, __FILE__, __LINE__);
                     }
                 }
+                        size_t total_chunksizes = 1;
+        for (const auto& chunk_size:d_chunksizes)
+            total_chunksizes *= chunk_size;
+        // If the chunk size is greater than 4M, we increase the chunk cache size to be 64M.
+        if (total_chunksizes  >4194304) {
```

Comment:
These might be better as constants so it's easier to see, at the top of the file, what the values are if we need to tune the parameters.